src/redux/dragons.js

```diff
@@ -7,9 +7,7 @@ export const fetchingDragonsApi = createAsyncThunk(
   async () => {
     const fetchedDragons = await axios
       .get('https://api.spacexdata.com/v3/dragons')
-      .catch((err) => {
-        console.log('error', err);
```

Comment:
Please kindly remove the `console.log` from your code. It's normally used for debugging and using it in the code is dangerous because not all browsers support `console.log` as it is not part of the standard and is an extension of the `DOM` thus you should not count on its presence and also using it can enable a client to debug the code and see all the client-side things that a developer would see.